modules/build/src/main/scala/scala/build/Project.scala

```diff
@@ -38,8 +38,14 @@ final case class Project(
       case (None, None) =>
         val baseJvmConf = bloopJvmPlatform
         baseJvmConf.copy(
-          // We don't pass jvm home here, because it applies only to java files compilation
-          config = baseJvmConf.config.copy(home = None)
+          // We don't pass jvm home to the general config
+          // it applies only to java files compilation
+          config = baseJvmConf.config.copy(home = None),
```

Comment:
It seems changing Java Home here makes Bloop actually use it while compiling, which breaks compatibility for when Bloop is being run on an older Java version (which isn't something we actively support, but neither should we actively break it if not strictly necessary)